CRM/Price/BAO/PriceField.php

```diff
@@ -376,21 +376,25 @@ public static function addQuickFormElement(
           $taxAmount = CRM_Utils_Array::value('tax_amount', $opt);
           if ($field->is_display_amounts) {
             $opt['label'] = !empty($opt['label']) ? $opt['label'] . '&nbsp;-&nbsp;' : '';
+            $pre_help = isset($opt['help_pre']) ?
+              '<span class="crm-price-amount-help-pre">' . $opt['help_pre'] . '</span>: ' : '';
+            $post_help = isset($opt['help_post']) ?
+              ': <span class="crm-price-amount-help-post">' . $opt['help_post'] . '</span>' : '';
```

Comment:
I think we need to also add `description` class to make the span look similar to other help texts(custom fields, etc).